### PR TITLE
HDDS-10116. Rename mockito2.version to mockito.version in pom.xml and fix typos

### PR DESCRIPTION
**Pull request by wzhallright**
## What changes were proposed in this pull request?
Mockito has been upgraded to 3.x,  but the `pom.xml` tag still named mockito2.version. Maybe it should be renamed to mockito3.version.And fixed some typos in this PR.
## What is the link to the Apache JIRA

[HDDS-10116](https://issues.apache.org/jira/browse/HDDS-10116)

## CI

https://github.com/wzhallright/ozone/actions/runs/7497056961

### CODE REVIEW

**Review by kerneltime — APPROVED** [review]

File: hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/server/http/TestRatisDropwizardExports.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
- * Test RatisDropwizardRexporter.
+ * Test RatisDropwizardExporter.
```
Comment (adoroszlai) [inline hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/server/http/TestRatisDropwizardExports.java:40]: Actually the class being tested is `RatisDropwizardExports`.

```suggestion
 * Test {@link RatisDropwizardExports}.
```

File: pom.xml [hunk 2]
```diff
@@ -217,7 +217,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <objenesis.version>1.0</objenesis.version>
     <okhttp.version>2.7.5</okhttp.version>
     <okio.version>3.6.0</okio.version>
-    <mockito2.version>3.5.9</mockito2.version>
+    <mockito3.version>3.5.9</mockito3.version>
```
Comment (adoroszlai) [inline pom.xml:220]: Well spotted.

This was originally named `mockito2.version` to distinguish from the version property for Mockito 1, which was also a dependency at the time.  I think now we can rename this to simply `mockito.version`.

**Review by adoroszlai — COMMENTED** [review]
Thanks @wzhallright for the patch.

File: pom.xml (hunk 2 above)
Comment (wzhallright) [reply pom.xml]: done

File: hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/server/http/TestRatisDropwizardExports.java (hunk 1 above)
Comment (wzhallright) [reply hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/server/http/TestRatisDropwizardExports.java]: done

**Pull request merged**